store/store_asserts.go

```diff
@@ -45,13 +45,6 @@ func (s *Store) assertionsEndpointURL(p string, query url.Values) *url.URL {
 }
 
 type assertionSvcError struct {
-	// v1 error fields
```

Comment:
we can't do this until the store switches to return us v2 endpoints in refresh fetch_assertions, we need to discuss that with Przemysław

Reply:
Ah, ok, thanks for pointing this out.